models/_generate_layers.py

```diff
@@ -345,6 +344,42 @@ def _parse_init_docstring(self):
   def __repr__(self):
     return f"<{self.__class__.__name__} {self.layer_class.__name__} {self.inspect_init_sig}>"
 
+  def get_super_call_code_str(self) -> str:
+    """
+    Inspects the given layer in RETURNN, extracts the super call from there, removes unwanted parameters.
+
+    :return: kwargs_repr
+      Python source code of kwargs for the super() call of the layer which are not excluded.
+      Such that `f"super().__init__({kwargs_repr})"` is valid code.
```

Comment:
Now this documentation is again not correct.

Reply:
updated it, is it better now? Or still to vague (not 100% how to formulate the documentation)?

Reply:
See new comment below.